ov/firmware_drivers.go

```diff
@@ -0,0 +1,239 @@
+package ov
+
+import (
+	"encoding/json"
+
+	"github.com/HewlettPackard/oneview-golang/rest"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"github.com/docker/machine/libmachine/log"
+)
+
+type parentBundle struct {
+	ParentBundleName string `json:"parentBundleName,omitempty"`
+	ReleaseDate      string `json:"releaseDate,omitempty"`
+	Version          string `json:"version,omitempty"`
+}
+
+type HotFixes struct {
+	HotfixName  string `json:"hotfixName,omitempty"`
+	ReleaseDate string `json:"releaseDate,omitempty"`
+	ResourceId  string `json:"resourceId,omitempty"`
+}
+
+type FWComponents struct {
+	ComponentVersion string          `json:"componentVersion,omitempty"`
+	FileName         string          `json:"fileName,omitempty"`
+	Name             string          `json:"name,omitempty"`
+	SwKeyNameList    []utils.Nstring `json:"swKeyNameList,omitempty"`
+}
+
+type FirmwareDrivers struct {
+	BaselineShortName     string              `json:"baselineShortName,omitempty"`
+	BundleSize            int                 `json:"bundleSize,omitempty"`
+	BundleType            string              `json:"bundleType,omitempty"`
+	Category              string              `json:"category,omitempty"`
+	Created               string              `json:"created,omitempty"`
+	Description           string              `json:"description,omitempty"`
+	ETAG                  string              `json:"eTag,omitempty"`
+	EsxiOsDriverMetaData  []utils.Nstring     `json:"esxiOsDriverMetaData,omitempty"`
+	FwComponents          []FWComponents      `json:"fwComponents,omitempty"`
+	Hotfixes              []HotFixes          `json:"hotfixes,omitempty"`
+	HpsumVersion          string              `json:"hpsumVersion,omitempty"`
+	IsoFileName           string              `json:"isoFileName,omitempty"`
+	LastTaskUri           string              `json:"lastTaskUri,omitempty"`
+	Locations             map[string]string   `json:"locations,omitempty"`
+	Mirrorlist            map[string][]string `json:"mirrorlist,omitempty"`
+	Modified              string              `json:"modified,omitempty"`
+	Name                  string              `json:"name,omitempty"`
+	ParentBundle          []parentBundle      `json:"parentBundle,omitempty"`
```

Comment:
please fix this to 
ParentBundle          parentBundle      `json:"parentBundle,omitempty"`